analytical/templatetags/google_analytics_gtag.py

```diff
@@ -0,0 +1,64 @@
+"""
+Google Analytics template tags and filters, using the new analytics.js library.
+"""
+
+from __future__ import absolute_import
+
+import re
+
+from django.template import Library, Node, TemplateSyntaxError
+
+from analytical.utils import (
+    disable_html,
+    get_required_setting,
+    is_internal_ip,
+)
+
+PROPERTY_ID_RE = re.compile(r'^UA-\d+-\d+$')
```

Comment:
If the ID should be `UA-XXXXXX-Y`, as you state below, then the regex should probably allow only a single number after the second dash, e.g. look like `r'^UA-\d+-\d$'`. Correct?

Reply:
As per the [Google documentation](https://support.google.com/analytics/answer/7372977?ctx=glossary), the second number is the property number. There is no explicit constraint that it has to be less than 10. As such, the property number may be larger than 9, and restricting to a single digit may be too restrictive.